app/ui-react/syndesis/src/modules/integrations/pages/detail/MetricsPage.tsx

```diff
@@ -105,7 +105,7 @@ export class MetricsPage extends React.Component {
                                                     parseInt(
                                                       metricsData.lastProcessed,
                                                       10
-                                                    )
+                                                    ) * 1000
```

Comment:
Let's do this on the server, so UI doesn't have to do `* 1000`

Reply:
Done.